packages/react/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,93 @@
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import {
+  Fragment,
+  type HTMLAttributes,
+  type ReactNode,
+  forwardRef,
+  useMemo,
+} from 'react';
+
+export type AvatarProps = {
+  /**
+   * The name of the person the avatar represents.
+   * Will be used to generate initials if no image is provided.
+   */
+  name?: string;
+  /**
+   * The color of the avatar.
+   *
+   * @default 'accent'
+   */
+  color?: 'accent' | 'neutral' | 'brand1' | 'brand2' | 'brand3';
+  /**
+   * The size of the avatar.
+   *
+   * @default 'md'
+   */
+  size?: 'xs' | 'sm' | 'md' | 'lg';
+  /**
+   * The shape of the avatar.
+   *
+   * @default 'circle'
+   */
+  variant?: 'circle' | 'square';
+  /**
+   * Image or icon to display inside the avatar.
+   *
+   * Gets `aria-hidden="true"`
+   */
+  children?: ReactNode;
```

Comment:
This is added to some components, some not - I guess here is nice to have it to have the specification shown in the documentation, but should we have it in all components or no? ☺️  Should we also add: "automatically shows initials if no children is provided"? ☺️

Reply:
We do this for components where we do something with the children, so we can explain it in the jsdoc.
I don't think it's fitting to add text about initials to the children prop, but rather the initials prop.